lumicks/pylake/tests/test_kymo.py

```diff
@@ -145,6 +145,33 @@ def test_plotting_with_force(h5_file):
         kymo.plot_with_force(force_channel="2x", color_channel="red")
         assert np.allclose(np.sort(plt.xlim()), [-0.5, 3.5], atol=0.05)
         assert np.allclose(np.sort(plt.ylim()), [10, 30])
+
+
+@cleanup        
+def test_plotting_with_histograms(h5_file):
```

Comment:
It's good that it tests whether the plot spins up and detects bad input, but I'm missing a test which actually tests the histogram data.